Dockerfile

```diff
@@ -1,19 +1,21 @@
+# hadolint ignore=DL3007
 FROM golang:latest as build
 WORKDIR /go/src/github.com/pomerium/pomerium
 
+# hadolint ignore=DL3008
 RUN apt-get update \
-    && apt-get -y install zip
+    && apt-get -y --no-install-recommends install zip
 
 # cache depedency downloads
 COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
 
 # build
-RUN make build-deps
-RUN make build NAME=pomerium
-RUN make build NAME=pomerium-cli
-RUN touch /config.yaml
+RUN make build-deps && \
```

Comment:
In this context, multiple layers are actually better.  If one of the three make commands is failing, I'd prefer to re-use the other layers rather than start from scratch at `make build-deps` every time.  This is a transient build stage so it doesn't make sense to optimize for space or pull performance.

Reply:
reverted in 3bb958f92366163fae3c64bf45149fa892cde0ab